src/nvim/tui/tui.c

```diff
@@ -1277,6 +1339,8 @@ void tui_flush(TUIData *tui)
     tui_busy_stop(tui);  // avoid hidden cursor
   }
 
+  tui_synchronize_output(tui, true);
```

Comment:
nit: the expansion of "synchronize" seems unusual
```suggestion
  tui_sync_output(tui, true);
```